modin/utils.py

```diff
@@ -42,7 +44,14 @@
 
 from pandas.util._decorators import Appender
 from pandas.util._print_versions import _get_sys_info, _get_dependency_info  # type: ignore[attr-defined]
-from pandas._typing import JSONSerializable
+from pandas._typing import (  # type: ignore[attr-defined]
```

Comment:
I don't know. Otherwise `mypy` was giving me strange errors on this import.

Anyway, after moving `walk_aggregation_dict` to the `modin.pandas.utils` this is not needed anymore.